### ISSUE TITLE
'rid release' and 'rid docs' fails if docs folder doesn't exist

### ISSUE THREAD

**Issue by Naycon** [rocjs/roc-internal-dev#6]
As the title says, the `rid release` and `rid docs` commands fail if the docs folder under `packages/roc-plugin-***` doesn't exist:

Output from running `rid release`

```
~/dev/roc-plugin-myplugin(branch:master) » npm run release

> roc-plugin-myplugin@ release /Users/johndoe/dev/roc-plugin-myplugin
> rid release

Next version (current version is 1.0.0)? 1.0.0
/Users/johndoe/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin/src/builder/index.js -> /Users/johndoe/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin/lib/builder/index.js
/Users/johndoe/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin/src/config/roc.config.js -> /Users/johndoe/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin/lib/config/roc.config.js
/Users/johndoe/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin/src/index.js -> /Users/johndoe/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin/lib/index.js
/Users/johndoe/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin/src/roc/index.js -> /Users/johndoe/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin/lib/roc/index.js
/Users/johndoe/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin/src/roc/util.js -> /Users/johndoe/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin/lib/roc/util.js

> roc-plugin-myplugin@1.0.0 lint /Users/johndoe/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin
> eslint .

  Warning    Configuration

  There was a mismatch in the application configuration structure, make sure this is correct.
  Did not understand packages

An error happened { [Error: ENOENT: no such file or directory, open '/Users/johndoe/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin/docs/Actions.md']
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/johndoe/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin/docs/Actions.md' }

npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/johndoe/.nvm/versions/node/v5.0.0/bin/node" "/Users/johndoe/.nvm/versions/node/v5.0.0/bin/npm" "run" "release"
npm ERR! node v5.0.0
npm ERR! npm  v3.3.6
npm ERR! code ELIFECYCLE
npm ERR! roc-plugin-myplugin@ release: `rid release`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the roc-plugin-myplugin@ release script 'rid release'.
npm ERR! This is most likely a problem with the roc-plugin-myplugin package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rid release
npm ERR! You can get their info via:
npm ERR!     npm owner ls roc-plugin-myplugin
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/johndoe/dev/roc-plugin-myplugin/npm-debug.log
------------------------------------------------------------
```

...and from running `rid docs`:

```
~/dev/roc-plugin-myplugin(branch:master) » npm run rid docs

> roc-plugin-myplugin@ rid /Users/johndoe/dev/roc-plugin-myplugin
> rid "docs"

  Warning    Configuration

  There was a mismatch in the application configuration structure, make sure this is correct.
  Did not understand packages

An error happened { [Error: ENOENT: no such file or directory, open '/Users/johndoe/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin/docs/Actions.md']
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/johndoe/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin/docs/Actions.md' }

npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/johndoe/.nvm/versions/node/v5.0.0/bin/node" "/Users/johndoe/.nvm/versions/node/v5.0.0/bin/npm" "run" "rid" "docs"
npm ERR! node v5.0.0
npm ERR! npm  v3.3.6
npm ERR! code ELIFECYCLE
npm ERR! roc-plugin-myplugin@ rid: `rid "docs"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the roc-plugin-myplugin@ rid script 'rid "docs"'.
npm ERR! This is most likely a problem with the roc-plugin-myplugin package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rid "docs"
npm ERR! You can get their info via:
npm ERR!     npm owner ls roc-plugin-myplugin
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/johndoe/dev/roc-plugin-myplugin/npm-debug.log
------------------------------------------------------------
~/dev/roc-plugin-myplugin(branch:master) » cd packages/roc-plugin-myplugin                                                                                                              johndoe@57128
------------------------------------------------------------
~/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin(branch:master) » mkdir rocs                                                                                                      johndoe@57128
------------------------------------------------------------
~/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin(branch:master) » rm rocs                                                                                                         johndoe@57128
rm: rocs: is a directory
------------------------------------------------------------
~/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin(branch:master) » rmdir rocs                                                                                                      johndoe@57128
------------------------------------------------------------
~/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin(branch:master) » mkdir docs                                                                                                      johndoe@57128
------------------------------------------------------------
~/dev/roc-plugin-myplugin/packages/roc-plugin-myplugin(branch:master) » cd ..                                                                                                           johndoe@57128
------------------------------------------------------------
~/dev/roc-plugin-myplugin/packages(branch:master) » cd ..                                                                                                                                 johndoe@57128
------------------------------------------------------------
~/dev/roc-plugin-myplugin(branch:master) » npm run rid docs                                                                                                                               johndoe@57128

> roc-plugin-myplugin@ rid /Users/johndoe/dev/roc-plugin-myplugin
> rid "docs"

  Warning    Configuration

  There was a mismatch in the application configuration structure, make sure this is correct.
  Did not understand packages

Done
```

Both issues were fixed by adding the `docs` folder to the `packages/roc-plugin-myplugin` folder.

**Comment by dlmr** [comment]
Thanks for the issue report, I will make sure this is fixed in the coming release.

**Comment by dlmr** [comment]
This has now been solved in `^2.0.0`.
